src/serializer.js

```diff
@@ -77,14 +79,8 @@ class FullPacketParser extends Transform {
           JSON.stringify(packet.data) + '; buffer :' + chunk.toString('hex'))
       }
     } catch (e) {
-      if (e.partialReadError) {
```

Comment:
Not sure why but for some reason PartialReadErrors were not being propagated up to the user here. Not the case for Parser either.